ddp-workspace/projects/ddp-angio/src/styles.scss

```
@@ -306,6 +306,35 @@ a:focus {
     color: mat-color($app-theme, 600);
 }
 
+.SimpleButton .ddp-globe {
+    stroke: mat-color($app-theme, 100);
+    fill: transparent;
+    vertical-align: middle;
+}
```

Can we add transition here and to other projects? Now animation looks a bit weird: text has smooth animation, but glode doesn't.
```suggestion
.SimpleButton .ddp-globe {
    stroke: mat-color($app-theme, 100);
    fill: transparent;
    vertical-align: middle;
    transition: stroke 0.3s ease;
}
```

---

Thanks for identifying this!  I could tell there was something different but I thought the SVG just changed colors faster.  This makes more sense!